ramls/examples/userSummaries.sample

```diff
@@ -0,0 +1,47 @@
+{
+  "userSummaries": [
+    {
+      "id": "54778f48-9d4d-488d-8bae-a3eaaf5b9d2a",
+      "outstandingFeeFineBalance": 10,
```

Comment:
Shouldn't this property have decimal value?

Reply:
updated